### PR TITLE
Explorer's run index table should support a mobile layout

### PR DESCRIPTION
**Pull request by NavyAdmiral**
For [#166275878](https://www.pivotaltracker.com/story/show/166275878)

### CODE REVIEW

File: explorer/client/src/components/JobRuns/List.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ interface IProps {
 }
 
 const HEADERS = ['Node', 'Run ID', 'Job ID', 'Created At']
+const MOBILE_HEADERS = ['Run ID', 'Job ID', 'Created At', 'Node']
```
Comment (rupurt) [inline explorer/client/src/components/JobRuns/List.tsx]: 👍

File: explorer/client/src/components/Table.tsx [hunk 2]
```diff
@@ -71,34 +77,48 @@ interface IProps extends WithStyles<typeof styles> {
   emptyMsg?: string
 }
 
+const renderCells = ({ r, idx }: { r: any[]; idx: number; }) => (
+  <TableRow key={idx}>
+    {r.map((col: Column, idx: number) => (
```
Comment (rupurt) [inline explorer/client/src/components/Table.tsx]: How do you feel about inlining this without `()`?

```jsx
{r.map((col: Column, idx: number) => <TableCell key={idx} column={col} />)}
```

Comment (NavyAdmiral) [reply explorer/client/src/components/Table.tsx]: I'd like it more inlined too, but linting forces the format.

File: explorer/client/src/components/Table.tsx [hunk 3]
```diff
@@ -71,34 +77,48 @@ interface IProps extends WithStyles<typeof styles> {
...
   } else {
     return rows.map((r: any[], idx: number) => (
-      <TableRow key={idx}>
-        {r.map((col: Column, idx: number) => (
-          <TableCell key={idx} column={col} />
-        ))}
-      </TableRow>
+      <>
+        <Hidden xsDown>{renderCells({ r, idx })}</Hidden>
+        <Hidden smUp>{renderCells({ r: [r[2], r[1], r[3], r[0]], idx: idx })}</Hidden>
```
Comment (NavyAdmiral) [inline explorer/client/src/components/Table.tsx]: @rupurt Would you like to see Constants for indices?

Comment (rupurt) [reply explorer/client/src/components/Table.tsx]: @NavyAdmiral I feel like this logic doesn't belong in `<Table>` as it's meant to be a generic component. We should be using `<Hidden>` to render different `<Table>` components in `JobRuns/List`.

**Comment by rupurt** [comment]
Closing in favor of #1285

**Pull request closed without merging**